src/app/menu/menu.component.html

```diff
@@ -0,0 +1,77 @@
+<h3>Main Menu</h3>
+<div class="menu">
+    <div class="col-lg-5 col-lg-offset-4">   
+        <a routerLink="/introduction/1">  
+            <img class="intro-hex" src="../../assets/img/intro.png">          
+            <button class="intro-btn">
+                Introduction                   
+            </button>
+        </a>
+        <div class="intro-outer-circle">
+            <div class="activity1-incomplete" *ngIf='stage<1 || (stage==1 && activity<1)'></div>
```

Comment:
There are too many values to pass to the directive (stage and activity fetched from the database + stage and activity of the representing div) and I feel this may not change the repetition.